aim/web/ui_v2/src/types/components/ChartPanel/ChartPanel.d.ts

```diff
@@ -8,9 +8,12 @@ import { IFocusedState } from '../../services/models/metrics/metricsAppModel';
 
 export interface IChartPanelProps {
   chartType: ChartTypeEnum;
-  data: ILine[][];
+  data: ILine[][] | any;
```

Comment:
Can you please remove type `any` from data and add exact interface and if it will be available to add in the future we can add FIXME or TODO comments to fix it later , thanks